ipaserver/plugins/service.py

```diff
@@ -201,6 +201,28 @@ def validate_realm(ugettext, principal):
         raise errors.RealmMismatch()
 
 
+def validate_auth_indicator(entry):
+    new_value = entry.get('krbprincipalauthind', None)
+    if not new_value:
+        return
+    # The following services are considered internal IPA services
+    # and shouldn't be allowed to have auth indicators.
+    # https://pagure.io/freeipa/issue/8206
+    pkey = api.Object['service'].get_primary_key_from_dn(entry.dn)
+    principal = kerberos.Principal(pkey)
+    servers = (s['cn'][0] for s in api.Command.server_find()['result'])
+    if principal.hostname in servers:
+        prefixes = ("host", "cifs", "ldap", "HTTP")
+    else:
+        prefixes = ("cifs")
+    if principal.service_name.startswith(prefixes):
```

Comment:
I think we want exact matches because this may catch other random principals that users might create. The service prefixes are basically arbitrary, as you can tell because of different casing, etc

So this instead:
if principal.service_name in prefixes: